did/plugins/sentry.py

```diff
@@ -112,8 +137,12 @@ def fetch(self):
         for activity in self.filter_data():
             if (activity['user']['email'] == self.user.email and
                     activity['type'] == 'note'):
-                self.stats.append("{0} - {1}".format(
-                    activity['issue']['shortId'], activity['issue']['title']))
+                record = "{0} - {1}".format(
+                    activity['issue']['shortId'], activity['issue']['title'])
+                # skip if the issue is in the stats already
+                if record in self.stats:
+                    continue
+                self.stats.append(record)
```

Comment:
What about introducing a common method `SentryStats.append()` which would handle uniqueness instead of copy-pasting the identical code?

Reply:
I was happy it works, so not much effort was put into refactoring. Thanks for this review, will do that.